src/TestLibrary/Nested/TestClass3.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
SA1633 - The file has no header, the header Xml is invalid, or the header is not located at the top of the file.